config/jobs/kubernetes/sig-cloud-provider/gcp/gce-conformance.yaml

```diff
@@ -37,6 +37,164 @@ periodics:
         requests:
           cpu: 1
           memory: 3Gi
+      env:
+      - name: ENABLE_APISERVER_ADVANCED_AUDIT
+        value: "true"
+      - name: ADVANCED_AUDIT_POLICY
```

Comment:
Can you comment why this is here?  In an ideal world, this job would be rewritten to patch the existing default policy .yaml, and explain why the patch.  Pretend you're a future maintainer trying to understand what this is, whether this is safe to remove, or when/why it needs to be changed.

Reply:
It seems to be the correct place to over-ride based on the underlying scripts.

https://github.com/kubernetes/kubernetes/blob/397466dec4c3f8cbf577cb7378a081fe9dad4c6c/cluster/gce/gci/configure-kubeapiserver.sh#L146-L150

Should we instead update the cmd/args of the job to run a series of commands to first run 'sed/awk' to patch / remove this, before we run the existing job:

https://github.com/kubernetes/kubernetes/blob/b1e3a2ac7a80d49ef6153457eaa35dac4fc98444/cluster/gce/gci/configure-helper.sh#L1178-L1182

Reply:
The patch, if we found a way to cleanly apply it within the job:

```
    diff default-policy.json new-policy.json 2>&1
```

```
  71,75d70
  <   # Don't log events requests.
  <   - level: None
  <     resources:
  <       - group: "" # core
  <         resources: ["events"]
```

Reply:
I updated the PR to only update the conformance job, and added a comment as to why we override the audit policy, including that it would be better to patch rather than override if possible.